_posts/2015-12-13-weak-tests-hurt.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title:  "Weak Tests Hurt"
+date:   2015-11-15 10:00:00
+categories: git development
+author: kent.safranski
+cover: /assets/images/covers/failed-test.jpg
```

Comment:
Haven't seen it yet, but sounds promising! Just food for thought on the image uploads. I've only seen it really hurt for sites with large amounts of video and audio.